tests/tests_pytorch/tuner/test_scale_batch_size.py

```diff
@@ -235,7 +234,7 @@ def test_error_on_dataloader_passed_to_fit(tmpdir):
 
     with pytest.raises(
         MisconfigurationException,
-        match="The batch scaling feature cannot be used with dataloaders passed directly",
+        match="Batch size finder cannot be used with dataloaders passed directly",
```

Comment:
Why drop the article 🤣 The poor articles. They are important!

Reply:
reverted